### ISSUE TITLE
Goods Screen - Poor Color Contrast for the Background Icons

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#122]
The color contrast ratio of the light gray goods icons ( #E5E5E5 ) on the white background ( #ffffff ) is 1.26:1 and needs to be 4.5:1 or higher to pass.

Also, why on the iOS version of Sweat and Toil did we change the background color ( orange ) of the labor types and the list of goods on the Goods screen change? It wasn't this way the last time I looked at the application, and it's not the same on the Android version.

**Comment by jfc3-dol** [comment]
For the iOS application, the color contrast of the black goods icon ( #000000 ) on the orange background ( #F09A37) passes with a color contrast ratio of 9.4:1. But the orange background ( #F09A37 ) on the white screen ( #ffffff ) does not. Its color contrast ratio is 2.2:1 and needs to be 4.5:1 or higher.

Why not remove the orange-colored background, and it will pass for everything.

Please see the attached screenshot.


![ilab - sweat and toil - ios - goods screen - icon color contrast issue](https://user-images.githubusercontent.com/10423487/88420225-91137400-cdb4-11ea-9eef-895cc53ec841.png)

The Android version passes with the black goods icons on the white screen background.

**Comment by jfc3-dol** [comment]
ILAB developers/team response below followed by my response.

Accessibility - iOS - Goods Screen - Goods Selector has Color Contrast Issue for the One Not Selected in Banner #126 

The goods icon orange background was used to display the goods icon in iOS normal and dark mode. If we remove this background our icons will not be visible in Dark mode. We’ll look at this issue and try to come up with a solution without removing color. You can discuss it with John and suggest the background color.

jfc iii response

Their issue response above is related to this issue and not the one they mentioned up above (#126 ). 

The Goods selector in the Banner section of the screen, not having enough color contrast. The icon issue is for the issue "Goods Screen - Poor Color Contrast for the Background Icons" #122.

For dark more, you could make it, so the background is black, and the icon goes to white instead of having the orange or at least add a black ( #000000 ) outline to the for the orange background, so there is enough color contrast between it and the white background it is on too.

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

The color contrast of the Goods icons for both light and dark mode pass, so I'm going to close this issue.
